docs/products/cassandra/index.rst

```diff
@@ -0,0 +1,19 @@
+Aiven for Apache Cassandra
+==========================
+
+What is Apache Cassandra?
+-------------------------
+
+`Apache Cassandra <https://cassandra.apache.org/_/index.html>`_ is an open source, NoSQL database. It is specifically designed to be highly available, performant, and scalable. It supports a distributed architecture with asynchronous replication and is highly fault tolerant.
+
+Apache Cassandra originated originally from Facebook, and since then has had a strong history with some of the most demanding data platforms around while being in the care of the Apache project. From this strong history comes a robust database with an emphasis on the security expected, and the ability to scale up to meet the needs of even the largest data platforms.
```

Comment:
"originated originally" feels a little repetitive - perhaps "came originally"?

I'd maybe simplify the first sentence as a whole - perhaps something like:
```
Apache Cassandra came originally from Facebook. It has had a strong history with some of the most demanding data platforms around.
```

I'd maybe also change "the security expected" to just "security"

Reply:
Or maybe "security expected" is meant to be "security aspect", which reads rather nicely.